docs/vaahcms/basic/api.md

```diff
@@ -11,6 +11,74 @@ Of course, you may always convert Eloquent models or collections to JSON using
 their toJson methods; however, Eloquent resources provide more granular and robust 
 control over the JSON serialization of your models and their relationships.
 
+### Sign In
+
+To get authorization token you need to `sign in` first. after the successfully `sign in`,
+an ` api_token ` in the response will be there which you can use as authorization token.
+
+For `sign in` follow below link :
+
+```php
+POST <public-url>/api/signin
+```
+##### Request samples
+```php
```

Comment:
@wri-abinash-s001 this should not be a php tag and it should be javascript and code should be something like:
```js
let params = {
  email: "",
  password: "",
}

```